File: pyg-shop-web/src/main/webapp/js/service/commentsService.js
app.service('commentsService',function($http){
    this.findAllByPage=function (currentpage,pagesize) {
        return $http.get("../comments/findBysellerid?currentPage="+currentpage+"&pageSize="+pagesize);
    }
    this.viewComments=function (id) {
        return $http.get("../comments/find?id="+id);
    }
    this.replyComments=function (id,content) {
        return $http.get("../comments/update?id="+id+"&content="+content);
    }
})

File: pyg-comments/pyg-comments-interface/src/main/java/top/ingxx/comments/service/CommentsService.java
package top.ingxx.comments.service;


import top.ingxx.entity.PageResult;
import top.ingxx.pojo.TbComments;


public interface CommentsService {
    public PageResult findAll(int currentPage, int pageSize);
    public void insertOneTbComments(TbComments tbComments);
    public PageResult queryTbCommentsBySpuId(String spuid, int currentPage, int PageSize);
    public PageResult queryTbCommentsByParentsId(String parentsid, int currentPage, int pageSize);
    public void  deleteOneTbComments(String id);
    public PageResult findByName(String sellerid, int currentPage, int pageSize);
    public TbComments find(String id);
    public void updateByid(String id,int content);
}
